conf/locale/fake2/LC_MESSAGES/django-partial.po

```diff
@@ -2,7 +2,7 @@
 # Copyright (C) 2015 EdX
 # This file is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
 # EdX Team <info@edx.org>, 2015.
-# 
+#
```

Comment:
These occurred because I upgraded my virtualenv's requirements - apparently Django was on an old version, and the new version strips spaces or something.